### PR TITLE
[RHACS] Added Kubernetes related guidence commands in the upgrade guide

### PR DESCRIPTION
**Pull request by gaurav-nelson**
For https://issues.redhat.com/browse/OSDOCS-2799

Preview: https://openshift-docs-git-osdocs-2799-fix-gnelson.vercel.app/openshift-acs/master/upgrading/upgrading_roxctl/upgrade-from-44.html

### CODE REVIEW

**Comment by gaurav-nelson** [comment]
@IAmATeaPot418 PTAL

**Comment by gaurav-nelson** [comment]
This PR only covers commands in the `roxctl` upgrade guide. I'll do a different PR for updating all commands with `oc`.

**Comment by IAmATeaPot418** [comment]
This seems a bit like overkill. It works but looks sloppy in comparison to how we previously managed the stackrox documentation.

File: modules/update-readiness-probes.adoc [hunk 1]
```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * upgrade/upgrade-from-44.adoc
+:_module-type: PROCEDURE
+[id="update-readiness-probes_{context}"]
+= Update readiness probes
+
+[role="_abstract"]
+If you are upgrading from {product-title} 3.65.0, you must run the following additional commands to update the readiness probe path.
```
Comment (kalexand-rh) [inline modules/update-readiness-probes.adoc:9]: s/commands/command (I only see one in this procedure.)

**Review by kalexand-rh — COMMENTED** [review]
This is looking good!

**Pull request merged**